ipn/ipnlocal/serve_test.go

```diff
@@ -560,6 +560,89 @@ func TestServeHTTPProxyHeaders(t *testing.T) {
 	}
 }
 
+func TestServeHTTPProxyQuery(t *testing.T) {
+	b := newTestBackend(t)
+
+	// Start test serve endpoint.
+	testServ := httptest.NewServer(http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			// Set the raw request query parameter to a response header, so
+			// the final output can can be checked in tests.
+			t.Logf("adding query parameters %s", r.URL.RawQuery)
+			w.Header().Add("QueryParam", r.URL.RawQuery)
+		},
+	))
+	defer testServ.Close()
+
+	tests := []struct {
+		name             string
+		inputQueryParams string
+		wantQueryParams  string
```

Comment:
just "in" and "want" is fine. You can say more in comments if needed.

But do you ever have a case where these two strings are different? Doesn't look like it?